test/CMakeLists.txt

```diff
@@ -37,9 +37,9 @@ ENDIF()
 
 
 ADD_EXECUTABLE(test_singa "gtest/gtest_main.cc" ${singa_test_source})
-ADD_DEPENDENCIES(test_singa singa)
+ADD_DEPENDENCIES(test_singa singa_static)
```

Comment:
There are two libraries here, singa_static and singa. In linux, there will be libsinga_static.a and libsinga.so.